### ISSUE TITLE
Syncing the position of screen reader and read aloud

### ISSUE THREAD

**Issue by ways2read** [daisy/reading-apps-ux-reqs#53]
Under "The user must be able to control the starting position of the read aloud", we currently say that if a screen reader is in use, read aloud should start from the screen reader position. Note this is a "should" clause inside a Must-have requirement.

No requirement says that when read aloud or embedded audio is stopped or paused, the screen reader focus should be placed at the last spoken position so the user can carry on reading from there.

Suggested addition to cover this gap:

> When read aloud is paused or stopped while a screen reader is in use, the screen reader reading position must be set to the text last spoken by the read aloud. The user must be able to continue reading with the screen reader from that point, or to review the preceding text, without having to search for their place.

**Comment by jylioja21** [comment]
It seems like a useful addition, but one concern is how difficult it is to implement. With the interaction of a reading system and a screen reader's virtual buffer, I'm not sure how easy it is to set focus to text that isn't a focusable item, without modifying the markup. Could we also list this as a should within a must, if it's found not to be easy to implement?
Another concern is that if I'm using web-based read aloud controls on the same page to pause or stop the read aloud, having screen reader focus jump from the controls to the document could be annoying. If this feature is added, it should be optional so the user can choose whether their screen reader focus gets changed.

**Comment by GeorgeKerscher** [comment]
In Thorium 3.5.1 with NVDA, you can do this. Press ctrl + shift +f10 and Thorium forceses a sync to the location where read aloud stopped. This same keystrok is how to get to the location of a bookmark and/or note.

**Comment by jylioja21** [comment]
Interesting. I was able to get this to work as well. An important point is that this is on demand with a keyboard command, so that it doesn't happen automatically when read aloud stops. This gives the choice of whether to change focus or not from the TTS controls, if I am using those to stop the read aloud. Doing it automatically if focus is in the content area may be an option as well, but I'd like to avoid a situation where focus is on the read aloud controls and is forced into the text area. This isn't a problem with Thorium's implementation, but we might want to include something about automatic focus management. Interestingly, with Thorium, if I have focus in the content area and pause read aloud with Control+2, it doesn't set screen reader focus to the item where tts stopped until I use the Control+Shift+F10 command.
